src/libcoretest/hash/mod.rs

```diff
@@ -18,6 +18,7 @@ struct MyHasher {
 }
 
 impl Default for MyHasher {
+    /// Constructs a `MyHasher` with initial value zero.
```

Comment:
I have nothing against adding documentation to tests, but is there any reason to do this?